src/main/java/slimeknights/tconstruct/gadgets/item/slimesling/BaseSlimeSlingItem.java

```diff
@@ -68,9 +68,19 @@ public void playerServerMovement(LivingEntity player) {
     }
   }
 
-  // Using function for consistency
+  protected void playSlingSound(PlayerEntity player, boolean isSuccess) {
+    if (isSuccess) {
+      playSuccessSound(player);
```

Comment:
Considering the only change is the pitch, this might be a bit too much abstraction. If you are just going to have two separate methods, just have two separate methods and ditch the one with the boolean parameter.